_includes/sidebar.html

```diff
@@ -1,16 +1,27 @@
+  <a href="https://minecraft.net/">Minecraft.net</a><br>
+  <a href="https://minecraft.net/realms">Minecraft Realms</a><br>
+  <a href="https://www.facebook.com/minecraft">Minecraft&nbsp;on&nbsp;Facebook</a><br>
+  <a href="https://findmcserver.com">Minecraft Server List</a><br>
+  <a href="https://shop.minecraft.net/">Merchandise</a><br><br>
+  <a href="https://bugs.mojang.com/browse/MC">Bug tracker</a><br>
+  <a href="https://help.minecraft.net/">Account Support</a><br><br>
+  <a href="http://twitter.com/Mojang">Mojang on Twitter</a><br>
+  <a href="http://twitter.com/MojangSupport">Support on Twitter</a><br><br>
+  <b>Try Mojang's other games!</b><br>
   <div style="height:5px; display:block;"></div>
-  <a href="http://scrolls.com/"><img src="http://assets.mojang.com/scrolls/scrolls_logo_150.png" border="0"></a>
+  <a href="http://scrolls.com/"><img src="http://assets.mojang.com/scrolls/scrolls_logo_150.png"></a>
   <div style="height:5px; display:block;"></div>
-  <a href="http://playcobalt.com/?utm_source=mcl"><img src="http://assets.mojang.com/cobalt/cobalt_logo_150.png" border="0"></a>
+  <a href="http://playcobalt.com/?utm_source=mcl"><img src="http://assets.mojang.com/cobalt/cobalt_logo_150.png"></a>
   <div style="height:5px; display:block;"></div>
-  <a href="https://www.minecraft.net/about-dungeons"><img src="{{ site.baseurl }}/img/logo-dungeons.png" border="0"></a>
+  <a href="https://www.minecraft.net/about-dungeons"><img src="{{ site.baseurl }}/img/logo-dungeons.png"></a>
   <div style="height:5px; display:block;"></div>
-  <a href="https://www.minecraft.net/about-legends"><img src="{{ site.baseurl }}/img/logo_legends.png" border="0"></a>
+  <a href="https://www.minecraft.net/about-legends"><img src="{{ site.baseurl }}/img/logo_legends.png"></a>
   <h2>Community links:</h2>
-  <a href="http://minecraftforum.net/">Minecraft&nbsp;Forums</a><br><a href="http://www.twitch.tv/directory/game/Minecraft">Minecraft on Twitch.tv</a><br><a href="https://minecraft.wiki">Minecraft Wiki</a>
+  <a href="http://minecraftforum.net/">Minecraft&nbsp;Forums</a><br>
+  <a href="http://www.twitch.tv/directory/game/Minecraft">Minecraft on Twitch.tv</a><br>
+  <a href="https://minecraft.wiki">Minecraft Wiki</a><br>
   <h2>OLauncher:</h2>
-  <a href="https://github.com/olauncher"><img src="{{ site.baseurl }}/img/olauncher_logo_150.png" border="0"></a>
```

Comment:
Your removal of `border="0"` causes borders to appear:
<details>
<summary>image</summary>
<img src="https://github.com/olauncher/olauncher.github.io/assets/21959175/e9df32b5-c281-4eeb-842b-cac20185f764" alt="OLauncher sidebar with borders" />
</details>